core/bessctl.cc

```diff
@@ -87,6 +87,28 @@ static inline Status return_with_errno(T* response, int code) {
   return Status::OK;
 }
 
+
+// This class is used as a resource manager to automatically pause workers if
+// running and then restarts workers if they were previously paused.
+class WorkerPauser {
+ public:
+  explicit WorkerPauser() {
+    if (is_any_worker_running()) {
+      workers_paused = true;
+      pause_all_workers();
```

Comment:
The other place in this file that calls `pause_all_workers()` also prints a log message.

I think the log message is useful, can you make sure it gets printed even in this case, perhaps by moving the log message inside pause_all_workers()?

Reply:
I wonder if we should use a conditional log, like VLOG(1) or something. I am concerned about cluttering the log file with this information. I am also expecting that this auto-pause could happen frequently and the LOG(INFO) call might prove to be expensive.

Reply:
You're right, it could get pretty verbose. VLOG(1) sounds good to me.